lib/main_common.pm

```diff
@@ -1078,7 +1078,8 @@ sub load_consoletests {
         loadtest "console/sshfs";
     }
     loadtest "console/mtab";
-    if (!get_var("NOINSTALL") && !get_var("LIVETEST") && (check_var("DESKTOP", "textmode"))) {
+    # disable these tests of server packages for SLED (poo#36436)
+    if (!get_var("NOINSTALL") && !get_var("LIVETEST") && (check_var("DESKTOP", "textmode")) && !is_desktop()) {
```

Comment:
not directly replated to this PR but still - 
do we really have such combinations that is_desktop will return true when DESKTOP=textmode ? 
something wrong with our naming of variables or how we use them :\

Reply:
and since you changing this line anyway you can remove extra brakets around `check_var('DESKTOP','textmode'');`

Reply:
Yes, the combination exists: see my link above: https://openqa.suse.de/tests/1794244 , this is command line testing for SLED.